examples/fitness/lib/settings.dart

```diff
@@ -4,6 +4,69 @@
 
 part of fitness;
 
+class _SettingsDialog extends StatefulComponent {
+  _SettingsDialogState createState() => new _SettingsDialogState();
+}
+
+class _SettingsDialogState extends State<_SettingsDialog> {
+  static final GlobalKey weightGoalKey = new GlobalKey();
```

Comment:
I'd probably make this non-static in case we ever want to show two instances of the dialog.